src/main/java/picard/fingerprint/CrosscheckFingerprints.java

```diff
@@ -77,14 +75,23 @@
             doc = "One or more input files (or lists of files) to compare fingerprints for.")
     public List<File> INPUT;
 
+    @Argument(shortName = "SI", optional = true, mutex={"MATRIX_OUTPUT"},
+            doc = "One or more input files (or lists of files) to compare fingerprints for. If this option is given " +
+                    "the program compares each sample in INPUT with the sample from SECOND_INPUT that has the same sample ID." +
+                    " In addition, data will be grouped by SAMPLE regardless of the value of CROSSCHECK_BY."+
+                    " When operating in this mode, each sample in " +
```

Comment:
This line is short.

Reply:
there are no newlines...so it doesn't matter.